src/owncloud/apache_config

```diff
@@ -14,3 +15,14 @@ Alias "/apps" "${SNAP_DATA}/owncloud/apps"
 <IfModule dir_module>
     DirectoryIndex index.html index.php
 </IfModule>
+
+# MPM Event
+<IfModule event.c>
+	StartServers			2
+	MinSpareThreads			4
+	MaxSpareThreads			32
+	ThreadsPerChild			16
+	ServerLimit				4
+	MaxRequestWorkers		64
+	MaxConnectionsPerChild	0
+</IfModule>
```

Comment:
Yeah let's remove this here. This rpi-specific stuff will probably need to go in the fork.